modules/update-service-install-web-console.adoc

```diff
@@ -18,20 +18,22 @@ Enter `Update Service` into the *Filter by keyword...* field to find the Operato
 
 . Choose *OpenShift Update Service* from the list of available Operators, and click *Install*.
 
-.. Channel `v1` is selected as the *Update Channel* since it is the only channel available in this release.
+.. Select an *Update channel*.
```

Comment:
I decided to turn this into a (potentially redundant) instruction because:

1. We're supposed to "[Use imperative mood in the first sentence of each step](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#writing-steps)" anyways.
2. Now we don't have to worry about updating this in the future if there ever is a second channel (not sure if that's likely but still, nice to avoid written commitments about "this release" wherever possible).
3. I have a feeling that some users will still click on the dropdown to verify that v1 is the only option anyways.